src/test/java/com/sleekbyte/tailor/utils/CharFormatUtilTest.java

```diff
@@ -88,4 +88,13 @@ public void testStartsWithAcronym() {
         assertFalse(CharFormatUtil.startsWithAcronym("xURLS"));
         assertFalse(CharFormatUtil.startsWithAcronym("shieldPROGRAMMEmarvel"));
     }
+
+    @Test
+    public void testBacktickEscapedIdentifier() {
+        // Backticks are not part of the identifier
```

Comment:
Maybe we could also test inputs like ``, single character strings, and empty strings to exercise all execution paths?